src/commands/synthetics/__tests__/mobile.test.ts

```diff
@@ -0,0 +1,16 @@
+import {promises as fs} from 'fs'
+import * as path from 'path'
+
+import * as mobile from '../mobile'
+
+// tslint:disable-next-line:no-var-requires
+const tmp = require('tmp-promise')
+
+describe('getMD5HashFromFileBuffer', () => {
+  test('correctly compute md5 of a file', async () => {
+    const dir = (await tmp.dir({mode: 0o755, unsafeCleanup: true})).path
+    await fs.writeFile(path.join(dir, 'file.txt'), 'Compute md5')
+    const fileBuffer = await fs.readFile(path.join(dir, 'file.txt'))
```

Comment:
Can we mock `fs.readFile` so that we don't need to introduce `tmp-promise` dependencies ?

Reply:
I think it's alright to add this depency if there are other tests based around file in the future, otherwise mocking `readFile` should do the trick yes

Reply:
I preferred to not mock because it was easy to reproduce.